mindmaps-test/src/test/java/io/mindmaps/test/graql/analytics/ScalingTestIT.java

```diff
@@ -455,34 +455,36 @@ public void testStatisticsWithConstantDegree() throws IOException, MindmapsValid
                     writer.println("starting with: " + workerNumber + " threads");
 
                     // configure assertions
-                    final int currentG = g;
+                    final long currentG = Long.valueOf(g);
+                    final long N = Long.valueOf(nodesPerStep);
+                    final long S = Long.valueOf(totalSteps);
                     statisticsAssertions.put(methods.get(0), number -> {
-                        Number sum = currentG * nodesPerStep * (1 + currentG * nodesPerStep + 6 * totalSteps * nodesPerStep + 2) / 2;
+                        Number sum = currentG * N * (1L + currentG * N + 6L * S * N + 2L) / 2L;
```

Comment:
Personally I am not at all a fan of shorthands. I.e. N = nodesPerStep, L = totalSteps. Just keep it as nodesPerStep in my opinion.